.circleci/config.yml

```diff
@@ -42,15 +42,17 @@ jobs:
           image: << parameters.image >>
   upload-images:
     docker:
-      - image: cimg/python:3.10.4
+      - image: cimg/base:2022.04
     resource_class: large
     steps:
       - prepare
+      - azure-cli/install
+      - run: sudo apt install python2
       - attach_workspace:
           at: .
       - run:
           name: Upload Sysroots
-          command: SKIP_SYSROOT_BUILD=true ./build/linux/sysroot_scripts/build_and_upload.py
+          command: SKIP_SYSROOT_BUILD=true python2 ./build/linux/sysroot_scripts/build_and_upload.py
```

Comment:
this script has `#!/usr/bin/env python3` at the top of it. can we drop the explicit `python2`?

Reply:
ah yes!